### PR TITLE
Add and subtract

### PR DESCRIPTION
**Pull request by ccbest**
add and subtract dunder methods
corresponding test

### CODE REVIEW

File: tests/test_precisenumbers.py [hunk 1]
```diff
@@ -159,3 +159,17 @@ def test_precisenumber_repr():
 def test_parse_number():
     with pytest.raises(NotImplementedError):
         precisenumbers.parse_number([1, 2])
+
+
+def test_add():
```
Comment (jessica-writes-code) [inline tests/test_precisenumbers.py:164]: Let's add some tests with non-zero stuff after the decimal.

File: precisenumbers/__init__.py [hunk 2]
```diff
@@ -146,3 +156,13 @@ def __str__(self) -> str:
...
 
         return f'{negative_indicator}{self.integer}.{"0" * (self.precision - len(str(self.fractional))) + str(self.fractional)}'
+
+    def __sub__(self, other):
+        """Addition"""
+        if not isinstance(other, PreciseNumber):
+            other = PreciseNumber(other)
+
+        return PreciseNumber(
+            float(self) - float(other),
```
Comment (jessica-writes-code) [inline precisenumbers/__init__.py:166]: I think this is gonna cause some weirdness. For example, 14 - 0.05 is going to equal 13. Is that what we want / what is dictated by sig-fig laws?

File: tests/test_precisenumbers.py (hunk 1 above)
Comment (ccbest) [reply tests/test_precisenumbers.py:164]: updated this test but one is wrong - let's discuss
